src/tls/README.md

```diff
@@ -0,0 +1,62 @@
+cockpit TLS proxy
+=================
+
+`cockpit-tls` is a TLS terminating HTTP proxy for cockpit-ws, with the (future)
```

Comment:
The emphasis you place on the client certificates matches the original intention of this project that you shared with me, but is no longer reflective of a world in which cockpit-tls is used by default.